apps/webhook/src/webhooks/usecases/webhook/webhook.usecase.ts

```diff
@@ -44,7 +57,19 @@ export class Webhook {
       throw new NotFoundException(`Provider with ${providerId} can not handle webhooks`);
     }
 
-    return await this.parseEvents(command, channel);
+    const events = await this.parseEvents(command, channel);
+
+    if (member) {
+      this.analyticsService.track('[Webhook] - Provider Webhook events parsed', member._userId, {
+        _organization: command.organizationId,
+        _environmentId: command.environmentId,
+        providerId,
+        channel,
+        events,
+      });
+    }
```

Comment:
I think actually that this one will be enough (no need to have the top one. wdyt?

Reply:
I added both to be able to track if we get more webhooks the we parse...